### PR TITLE
Enable sl micro 60

### PR DESCRIPTION
**Pull request by mcalmer**
## What does this PR change?

Prepare bootstrap repo and bootstrapping code for SL-Micro 6.0

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
- No tests: already covered

- [x] **DONE**

## Links

Issue(s): https://github.com/SUSE/spacewalk/issues/23758
Port(s): # 

- [ ] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by mackdk — APPROVED** [review]

File: spacewalk/certs-tools/spacewalk-certs-tools.changes.mc.enable-sl-micro-60 [hunk 1]
```diff
@@ -0,0 +1 @@
+- calculate bootstrap repo path for SL-Micro 6
```
Comment (vzhestkov) [inline spacewalk/certs-tools/spacewalk-certs-tools.changes.mc.enable-sl-micro-60:1]: Bit nitpicky, but maybe change to `SL Micro 6.0` ?

File: susemanager-utils/susemanager-sls/susemanager-sls.changes.mc.enable-sl-micro-60 [hunk 2]
```diff
@@ -0,0 +1,2 @@
+- define bootstrap repo data for SL-Micro 6.0
+- calculate bootstrap repo path for SL-Micro 6
```
Comment (vzhestkov) [inline susemanager-utils/susemanager-sls/susemanager-sls.changes.mc.enable-sl-micro-60:2]: Same, here, not sure about `SL-Micro` or `SL Micro` but there was mentioned somewhere `6.0`, but not `6`

File: utils/spacewalk-utils.changes.mc.enable-sl-micro-60 [hunk 3]
```diff
@@ -0,0 +1 @@
+- add SL-Micro 6.0 Uyuni Client Tools Channel via spacewalk-common-channels
```
Comment (vzhestkov) [inline utils/spacewalk-utils.changes.mc.enable-sl-micro-60:1]: And here.

**Review by vzhestkov — APPROVED** [review]
Very nitpicky comments for the changelog entries, but other than that looks good to me :+1:

File: spacewalk/certs-tools/spacewalk-certs-tools.changes.mc.enable-sl-micro-60 (hunk 1 above)
Comment (mcalmer) [reply spacewalk/certs-tools/spacewalk-certs-tools.changes.mc.enable-sl-micro-60]: changed

File: susemanager-utils/susemanager-sls/susemanager-sls.changes.mc.enable-sl-micro-60 (hunk 2 above)
Comment (mcalmer) [reply susemanager-utils/susemanager-sls/susemanager-sls.changes.mc.enable-sl-micro-60]: changed

File: utils/spacewalk-utils.changes.mc.enable-sl-micro-60 (hunk 3 above)
Comment (mcalmer) [reply utils/spacewalk-utils.changes.mc.enable-sl-micro-60]: changed

**Comment by admd** [comment]
@mcalmer a unit test seems to be failing.

**Comment by mcalmer** [comment]
> @mcalmer a unit test seems to be failing.

@admd This PR has no java changes and this test break also when it run on master directly.
@mackdk wanted to check later what happened. Maybe junit was updated and now break something.

**Comment by admd** [comment]
Ok, in that case, I will merge the PR. Thank you.

**Pull request merged**